packages/babel-plugin-transform-typescript/README.md

```diff
@@ -7,14 +7,14 @@ Does not type-check its input. For that, you will need to install and set up Typ
 ## Caveats
 
 * Does not support [`namespace`][namespace]s.
-* Does not support [`const enum`][const_enum]s because those require type information to transpile.
-* Does not support [`export =`][exin] and [`import =`][exin], because those cannot be transpiled to ES.next.
+* Does not support [`const enum`][const_enum]s because those require type information to compile.
+* Does not support [`export =`][exin] and [`import =`][exin], because those cannot be compile to ES.next.
 
 ## Workarounds
 
-* `namespace`: Migrate to using the `module { }` syntax instead.
+* `namespace`: Move to using [file exports][fm], or migrate to using the `module { }` syntax instead.
```

Comment:
Thoughts on combining the workaround next to its caveat?

Reply:
works for me